### ISSUE TITLE
English demo dataset

### ISSUE THREAD

**Issue by sverhoeven** [ADAH-EviDENce/evidence#47]
As I a non-Dutch speaking researcher I would like to evaluate the tool, but the supplied demonstration dataset is in Dutch. Can a English demonstration dataset be added?

**Comment by meiertgrootes** [comment]
I guess, o, although I don't have one. Equally problematic in this regard, is that the UI dialogue is in dutch ...

**Comment by sverhoeven** [comment]
You are right, created https://github.com/ADAH-EviDENce/WorkingTitleCloseReader/issues/48 for UI.

**Comment by sverhoeven** [comment]
For training the framework/code/tokenizer.ipynb is now using Dutch stopwords. It should be replaced with a English stop word list.
